downstream/modules/platform/proc-control-data-collection.adoc

```diff
@@ -1,20 +1,17 @@
-
-
 [id="proc-controlling-data-collection_{context}"]
 
 = Controlling data collection from {ControllerName}
 
-
 [role="_abstract"]
-You can control how {ControllerName} collects data by setting your participation level in the **User Interface** tab in the settings menu.
+You can control how {ControllerName} collects data by setting your participation level in the *User Interface* tab in the *Settings* menu.
 
 .Procedure
 
-. Log in to your {ControllerName}
-
-. Navgate to *Settings* -> *User Interface*
-. Select the desired level of data collection from the User Analytics Tracking State drop-down list:
-.. *Off*: Prevents any data collection.
-.. *Anonymous*: Enables data collection without your specific user data.
-.. *Detailed*: Enables data collection including your specific user data.
+. Log in to your {ControllerName}.
+. Navigate to *Settings > User Interface*.
```

Comment:
Used *Settings > User Interface* here because the `menu:` asciidoc tag renders the menu items as item1 -> item2.
Which is better from an accessibility viewpoint?

Reply:
@ariordan-redhat From an accessibility standpoint, it's better to use the menu:item1[item2]. The rendering is something I've been trying to push DXP to fix but it keeps getting deferred. I wonder if we should discuss this again in the next team meeting?